jbehave-core/src/main/java/org/jbehave/core/model/ExamplesTableFactory.java

```diff
@@ -95,6 +95,14 @@ public ExamplesTable createExamplesTable(String input) {
             String loadedTable = resourceLoader.loadResourceAsText(tableAsString.trim());
             tablePropertiesQueue = tableParsers.parseProperties(loadedTable);
             Deque<TableProperties> target = tablePropertiesQueue.getProperties();
+
+            boolean hasTransformers = target.getFirst().getTransformer() != null;
+            if (hasTransformers) {
```

Comment:
please update docs to describe this behavior: https://jbehave.org/reference/stable/tabular-parameters.html -> Using table transformers